CHANGELOG.txt

```diff
@@ -65,3 +65,5 @@
     - Add resumable support to operations
 2.2.1:
     - Fix bug when delete EC2 instance
+2.2.2:
+    - Add new EBS relationship to attach ebs volume to EC2 instance
```

Comment:
Add message about changing in handling on delete (issue 128 and issue 129)

Reply:
Can you elaborate  more on this ?